sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -5,7 +5,7 @@
 # --------------------------------------------------------------------------
 # pylint: disable=too-few-public-methods, too-many-instance-attributes
 # pylint: disable=super-init-not-called, too-many-lines
-
+import urllib.parse
```

Comment:
nit: Can you change this to `from urllib.parse import unquote` and change the usages below to just `unquote(`? Just looks cleaner that way.